app/models/Team.tsx

```diff
@@ -0,0 +1,32 @@
+import { ITeam } from '../utils/wikiQuery'
+
+export default class Team {
+    teamName: string
+    relationship: string
+    isParticipating: boolean
+    eliminationOrder: number
+
+    constructor(inTeam: ITeam) {
+
+        if ((inTeam.eliminationOrder === 0 && !inTeam.isParticipating) ||
+            (inTeam.eliminationOrder !== 0 && inTeam.isParticipating)) {
+            console.warn("Building a team whose eliminationOrder is 0 (default), but they are also have isParticipating = false. May be a bad team construction")
```

Comment:
We might want to include something to identify this possibly invalid team in the warning.

Reply:
That's a good suggestion. Any idea how to do that? Do you think we can rely on `teamName` here?

Reply:
`teamName` might give more information if something were going wrong, but this might warrant an additional `id` field that would be unique for each Team object, offering insights into potential issues with duplicate teams, incomplete teams, and empty teams.

Reply:
Exactly! I would go so far as to say that this seems fairly intertwined with #25